index.html

```diff
@@ -1,5 +1,6 @@
 <html>
 <body>
 <div>City Bangalore</div>
+<div>Country India</div>
```

Comment:
India is a Country, You might to relocate to other country if you want to move to foreign